### DISCUSSION TITLE
Who hosted the 🎥🏆 in 1994, 1996, 1999, and 2002?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#887] [Galaxy Trivia] [answered]
**Bonus:** Who holds the record for hosting the most 🎥🏆?

**Comment by peterbaccaro** [comment]
**Whoopi Goldberg** hosted the Oscars in 1994, 1996, 1999, and 2002.

The person who has triumphed more times at the Oscars than anyone else in history is **Walt Disney**. America's most iconic filmmaker racked up a whopping 26 Oscars, four of which were honorary awards.

**Reply by ParanoidUser** [reply]
How about the person who has hosted the most 🎥🏆?

**Reply by peterbaccaro** [reply, marked as answer]
Actor and comedian **Bob Hope** holds the record for the most frequent Oscar host with 19 appearances either solo or as co-host.
